exporter_test.go

```diff
@@ -613,48 +613,6 @@ version = "4.5.6"
 				h.AssertEq(t, fakeAppImage.IsSaved(), true)
 			})
 
-			when("image has a digest identifier", func() {
```

Comment:
Pulled these out of the `when("previous image exists", ...` context since the assertions aren't dependent on a previous image.